source/extensions/filters/network/dubbo_proxy/active_message.h

```diff
@@ -78,6 +78,8 @@ class ActiveMessageFilterBase : public virtual DubboFilters::FilterCallbacksBase
   Event::Dispatcher& dispatcher() override;
   void resetStream() override;
 
+  bool dual_filter() const { return dual_filter_; }
```

Comment:
no need

Reply:
it is not a friend class，this method is needed.

Reply:
hmm I thought you made it friend https://github.com/envoyproxy/envoy/blob/e442ba565c948fb2d6c9d037ce12eaecbb75b252/source/extensions/filters/network/dubbo_proxy/active_message.h#L218 but if it's not the case, make it friend, or name this `dualFilter`

Reply:
done, i modified it to friend.